src/lxp/packet.rs

```diff
@@ -538,6 +538,8 @@ pub enum Register {
     AcChargeEndSocLimit = 161,   // SOC at which AC charging will end (%)
 }
 
+pub const NUM_HOLDING_REGISTERS: u16 = 178; // that we know of
```

Comment:
from `LXP_REGISTERS.txt`. I don't know how accurate this is.

Reply:
I dare say there are some missing. Most of that list was from sight of an (old) protocol doc and some reverse engineering of the Android Luxpower app ;)

It'll vary depending on inverter model/firmware too no doubt. But until someone says otherwise I'm sure this is fine.

Reply:
Since I'm thinking about this in terms of pages of 40 values, I don't need this constant any more.